nginx/conf.d/default.conf

```diff
@@ -6,11 +6,6 @@ server {
     location / {
         proxy_pass         http://canopas-website-frontend:3080;
         proxy_set_header   Host $host;
-        proxy_set_header   X-Forwarded-For $proxy_add_x_forwarded_for;
```

Comment:
Why do we need to remove these headers? especially `X-Forwarded-For`? Isn't it will be used for forwarding?

Reply:
We don't have any use of these headers. Reverse proxy just works with proxy_pass attribute.